openedx/core/djangoapps/organization_api/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+URLs for the organization app.
+"""
+from django.conf.urls import patterns, url
+
+from openedx.core.djangoapps.organization_api.api import views
+
+
+ORGANIZATION_KEY_PATTERN = r"(?P<organization_key>((?![\^'\!\(\)\*\s]).)*)"
```

Comment:
Actually this was added after discussion with the solutions team. The organizations are added from the django admin and there is no validation there like we have for the organization in `studio` so I am going to remove this validation.